client/src/templates/Introduction/intro.css

```diff
@@ -58,7 +58,7 @@ a.cert-tag:active {
 }
 
 .block-grid .challenge-jump-link a:focus:not(:focus-visible) {
-  outline: none;
+  outline-color: transparent;
```

Comment:
```suggestion
```

Revert back to `outline: none`. This is for non-keyboard focus in which we don't want a focus outline.